packages/es-components/src/components/controls/checkbox/Checkbox.js

```diff
@@ -4,20 +4,39 @@ import styled, { withTheme } from 'styled-components';
 
 import { Label } from '../BaseControls';
 
-/* eslint-disable no-confusing-arrow */
+const backgroundColorSelect = (checked, theme, validationState) => {
+  if (checked) {
+    if (validationState === 'default') {
```

Comment:
I wonder if there is a way we can reference this string from some sort of constant. It would be nice to be able to update this as `validationState === validationStates.DEFAULT` or something along those lines.

I have some mixed feelings about the nested ifs. It's purely a style choice at this point, but we could at least simplify the `if (checked)` body to `return validationState === 'default' ? theme.colors.primary : theme.colors[validationState];`

Reply:
I'll replace the nested if with the ternary. Originally I avoided using a nested ternary due to lint warnings, but just went all-in on the ifs. haha

I wouldn't mind having a const for the validationState, but I'd also want to do it for the other states for consistency. They're all defined in the propTypes in string form currently and only used there, so I also don't want to add a bunch of definitions and only end up using one of them a couple of times... if that makes sense.

Reply:
Makes perfect sense. If we extract the validation states for Radio and Checkbox into their own constant we can clean up some of these magic strings. While I want this to be a worthwhile change I don't know how well it will play with the props section of the styleguide.